### ISSUE TITLE
Save MapSettings to the system

### ISSUE THREAD

**Issue by CoolCapri** [alaurenz/metrobike#126]
Currently we are not saving MapSettings (Such as map type) so that users will always get the same type of map (the default one) and other settings once they exit and re-launch MetroBike. We should save them on the exit of our app.

**Comment by CoolCapri** [comment]
There is also a bug only to be fixed once this feature is finished. You can select any map type for main activity, and it will be activated immediately. But when you restart the map, everything on the map will be recreated, although map type shown in Settings is still the map type of the previous map. This is also mentioned in the code.

**Comment by sw11** [comment]
True! I just fix the main activity page.
I will hopefully fix the setting consistency before beta

**Comment by sw11** [comment]
Right now the app will remain the same status if exit or re-launch.
But NOT killed. It requires to save data into file and I will work on that in version 1.

**Comment by CoolCapri** [comment]
Also, it seems that when you are in ResultsActivity (not sure about NavActivity), if you change settings to enable traffic information, it will not immediately update that content unless you refresh it. This is not a big deal but worth mentioning, though.

**Comment by CoolCapri** [comment]
@sw11 Now the only issue left here is that, when you exit the app, COMPLETELY CLOSE IT IN APP MANAGER and re-enter the app, the map type and other status will be lost.

**Comment by dutchscout** [comment]
This behavior seems acceptable to me, as long as history is preserved.  What do others think?

**Comment by sw11** [comment]
I won't fix the map type issue in this phrase. Maybe I will save the map type status in the next phrase if we have one.
